docs/components/carousel.html

```diff
@@ -1,112 +1,227 @@
+        </div>
+        <div class="carousel-item col-12 col-md-3">
+          <img class="img-fluid mx-auto d-block" src="//via.placeholder.com/600x400?text=2" alt="slide 2"/>
+        </div>
+        <div class="carousel-item col-12 col-md-3">
+          <img class="img-fluid mx-auto d-block" src="//via.placeholder.com/600x400?text=3" alt="slide 3"/>
+        </div>
+        <div class="carousel-item col-12 col-md-3">
+          <img class="img-fluid mx-auto d-block" src="//via.placeholder.com/600x400?text=4" alt="slide 4"/>
+        </div>
+        <div class="carousel-item col-12 col-md-3">
+          <img class="img-fluid mx-auto d-block" src="//via.placeholder.com/600x400?text=5" alt="slide 5"/>
+        </div>
+        <div class="carousel-item col-12 col-md-3">
+          <img class="img-fluid mx-auto d-block" src="//via.placeholder.com/600x400?text=6" alt="slide 6"/>
+        </div>
+        <div class="carousel-item col-12 col-md-3">
+          <img class="img-fluid mx-auto d-block" src="//via.placeholder.com/600x400?text=7" alt="slide 7"/>
+        </div>
+        <div class="carousel-item col-12 col-md-3">
+          <img class="img-fluid mx-auto d-block" src="//via.placeholder.com/600x400?text=8" alt="slide 8"/>
+        </div>
+      </div>
+      <a class="carousel-control carousel-control-prev carousel-control--background carousel-control--loop" href="#carouselLoop4" role="button" data-slide="prev">
+        <span class="carousel-control-prev-icon sni sni__chevron-prev justify-content-center align-items-center" aria-hidden="true"></span>
+        <span class="sr-only">Previous</span>
+      </a>
+      <a class="carousel-control carousel-control-next carousel-control--background carousel-control--loop" href="#carouselLoop4" role="button" data-slide="next">
+        <span class="carousel-control-next-icon sni sni__chevron-next justify-content-center align-items-center" aria-hidden="true"></span>
+        <span class="sr-only">Next</span>
+      </a>
+    </div>
 
-                <a class="carousel-control carousel-control-prev carousel-control--background carousel-control--right-lg" href="#carouselWithGroupedControls" role="button" data-slide="prev">
-                    <span class="carousel-control-prev-icon sni sni__chevron-prev justify-content-center align-items-center" aria-hidden="true"></span>
-                    <span class="sr-only">Previous</span>
-                </a>
-                <a class="carousel-control carousel-control-next carousel-control--background carousel-control--right-lg" href="#carouselWithGroupedControls" role="button" data-slide="next">
-                    <span class="carousel-control-next-icon sni sni__chevron-next justify-content-center align-items-center" aria-hidden="true"></span>
-                    <span class="sr-only">Next</span>
-                </a>
-            </div>
+    <h1 class="mt-5">CAROUSEL LOOP 4 ITEMS CENTERED</h1>
+    <div class="carousel-loop-centered position-relative">
```

Comment:
carousel--loop and carousel--centred should be separated. We might also have a carousel centred without loop.
Also this class should go along with the main carousel item, not as parent.